lib/fabrik/actions/JSExecuter.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const Promise = require('bluebird');
+const action = process.argv[2];
+const phase = process.argv[3];
+let context = process.argv[4];
+
```

Comment:
Do we want to add some validation here with regards to required args (atleast action/phase) and throw a meaningful error rather than dumping some JS trace which would say not defined etc..?